src/pytesseract.py

```diff
@@ -192,6 +191,20 @@ def file_to_dict(tsv, cell_delimiter, str_col_idx):
     return result
 
 
+def osd_to_dict(osd):
+    osd_keys = {
```

Comment:
Since the osd_keys dict is a simple constant mapping,
we can extract it outside of the function as a standard python constant (for example - OSD_KEYS).

Reply:
Yes it's a constant, we could do like that.